Packs/QualysFIM/Integrations/QualysFIM/QualysFIM_test.py

```diff
@@ -210,13 +267,45 @@ def test_fetch_incidents_command(requests_mock) -> None:
      - Ensure a sample value from the API matches what is generated in the context.
      - Ensure occurred time is correct.
     """
-    from QualysFIM import Client, fetch_incidents
+    from QualysFIM import fetch_incidents
+
     mock_response = util_load_json('test_data/fetch_incidents.json')
     requests_mock.post(f'{BASE_URL}fim/v3/incidents/search', json=mock_response)
-    requests_mock.post(f'{BASE_URL}/auth', json={})
-    client = Client(base_url=BASE_URL, verify=False, proxy=False, auth=('a', 'b'))
-    next_run, incidents = fetch_incidents(client=client, last_run={}, fetch_filter='',
-                                          first_fetch_time='3 days', max_fetch='2')
+
+    _, incidents = fetch_incidents(client=qualysfim_client, last_run={}, fetch_filter='',
+                                   first_fetch_time='3 days', max_fetch='2')
     raw_json = json.loads(incidents[0].get('rawJSON'))
+
     assert raw_json.get('id') == '75539bfc-c0e7-4bcb-b55a-48065ef89ebe'
     assert raw_json.get('createdBy').get('date') == 1613378492427
+
+
+def test_create_event_or_incident_output() -> None:
+    """
+    Given:
+        - A list of table headers and a QualysFIM API event
+    When:
+        - create_event_or_incident_output is called
+    Then:
+        - Assert correct output dictionary and no empty values
+    """
+    from QualysFIM import create_event_or_incident_output
+
+    # Set
+    table_headers = ['id', 'severity', 'dateTime', 'agentId', 'fullPath']
+    raw_event: dict = util_load_json('test_data/get_event.json')
+
+    # Arrange
+    event_output = create_event_or_incident_output(raw_event, table_headers)
+
+    # Assert
+    assert 'customerId' not in event_output
+    assert event_output == {
+        'id': raw_event['id'],
+        'severity': raw_event['severity'],
+        'dateTime': raw_event['dateTime'],
+        'agentId': raw_event['asset']['agentId'],
+        'fullPath': raw_event['fullPath'],
+    }
+    # function calls remove_empty_elements
+    assert all(value != {} or value != [] for value in event_output.values()), 'One or more event values are empty'
```

Comment:
```suggestion
    assert {[], {}}.isdisjoint(event_output.values()), 'One or more event values are empty'
```

Reply:
`set.isdisjoint()` won't work since the elements of the set (list and dict) are _**not hashable**_